task/06-conditions-n-loops-tasks.js

```diff
@@ -356,7 +499,13 @@ function timespanToHumanString(startDate, endDate) {
  *    365, 10 => '365'
  */
 function toNaryString(num, n) {
-    throw new Error('Not implemented');
+     
+    let res = '';
+    while(num >= n){
+        res = num % n + res;
+        num = Math.floor(num/n);        
+    }
+    return (num + res).toString();
```

Comment:
I meant using toString instead of your logic.
![Selection_006](https://user-images.githubusercontent.com/26328278/91644121-ca897000-ea41-11ea-9334-8d3ba6adb2cb.png)